### PR TITLE
Minor changes to the FAQ

### PR DESCRIPTION
**Pull request by periclesrocha**
Minor updates to the FAQ section

### CODE REVIEW

**Comment by periclesrocha** [comment]
@WilliamDAssafMSFT this is the PR I referred to. Let me know if this looks good, please. It's a very small change.

File: docs/t-sql/statements/copy-into-transact-sql.md [hunk 1]
```diff
@@ -430,9 +430,9 @@ WITH (
 ## FAQ
 
 ### What is the performance of the COPY command compared to PolyBase?
-The COPY command will have better performance depending on your workload. For best loading performance, consider splitting your input into multiple files when loading CSV. This guidance applies to gzip compressed files as well.
+The COPY command will have better performance depending on your workload. For best loading performance, consider splitting your input into multiple files when loading compressed CSV.
```
Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/copy-into-transact-sql.md]: This feels like a net loss of valuable information for customers. Does this guidance not apply to gzip compressed files? Should we specify the types of compression specified?

**Comment by Jak-MS** [comment]
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/copy-into-transact-sql.md (hunk 1 above)
Comment (periclesrocha) [reply docs/t-sql/statements/copy-into-transact-sql.md]: @WilliamDAssafMSFT let me know if you are satisfied with this PR based on the reply from Sumin in our e-mail thread.

Comment (WilliamDAssafMSFT) [reply docs/t-sql/statements/copy-into-transact-sql.md]: I see that. Should we add "Manually splitting files is needed only for compressed CSV files."?

This guidance is still a little confusing to me as someone who used to design a lot of ETL systems. Are we saying that a large uncompressed file wouldn't need to be split, but a smaller compressed file should be split up by the data source?

Comment (WilliamDAssafMSFT) [inline docs/t-sql/statements/copy-into-transact-sql.md]:
```suggestion
The COPY command will have better performance depending on your workload. 
- Uncompressed files can be split automatically, so there's no need to manually split CSV files. 
- Large compressed cannot be split automatically. For best loading performance, consider splitting your input into multiple files when loading compressed CSV.
```

@periclesrocha above ok with you?

**Comment by periclesrocha** [comment]
Thanks @WilliamDAssafMSFT! This looks good. I'll proceed with the PR

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Comment by periclesrocha** [comment]
#sign-off

**Pull request merged**